### PR TITLE
feat: 文件流下载完成

### PR DESCRIPTION
**Pull request by architectMymine**
### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [ ] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).

### CODE REVIEW

**Comment by architectMymine** [comment]
yes

**Comment by architectMymine** [comment]
get it

File: src/layouts/default/sidebar/components/Menu.vue [hunk 1]
```diff
@@ -37,6 +37,9 @@
       <a-menu-item key="11" @click="handleClick('/demo/fullscreen')">
         全屏
       </a-menu-item>
+      <a-menu-item key="12" @click="handleClick('/demo/file-download')">
```
Comment (alexzhang1030) [inline src/layouts/default/sidebar/components/Menu.vue:40]: 现在已经不需要手动引入路由了

File: src/router/routes/modules/demo.ts [hunk 2]
```diff
@@ -46,6 +46,11 @@ const route: RouteRecordRaw = {
       path: 'fullscreen',
       name: 'fullscreen',
       component: () => import('~/views/demo/fullscreen/index.vue')
+    },
```
Comment (alexzhang1030) [inline src/router/routes/modules/demo.ts:49]: 这里记得配一下 route.meta

File: src/utils/file/download.ts [hunk 3]
```diff
@@ -1,45 +1,14 @@
-import { openWindow } from '..'
 
-/**
- * Download file according to file address
- * @param {*} sUrl
- */
-export function downloadByUrl({
```
Comment (alexzhang1030) [inline src/utils/file/download.ts:7]: 这个函数你写的吗还是之前的，请确保一下删除对核心逻辑不会产生问题

File: src/views/demo/file-download/index.vue [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  </a-card>
+  <a-card title="图片URL下载示例">
+    <a-button type="primary">
+      下载
+    </a-button>
+  </a-card>
+</template>
+
+<script lang="ts" setup>
+import { downloadDatafile } from '~/utils/file/download'
```
Comment (alexzhang1030) [inline src/views/demo/file-download/index.vue:30]: 样例是不是还没写完，感觉不是很完善的样子

File: src/utils/file/download.ts (hunk 3 above)
Comment (architectMymine) [reply src/utils/file/download.ts:7]: 之前的没要了，现在的是我自己写的。

File: src/views/demo/file-download/index.vue [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+const handleDownloadData = () => {
+  downloadDatafile('text content', 'testFile.txt')
+}
+</script>
+
+<style>
```
Comment (likui628) [inline src/views/demo/file-download/index.vue:37]: 建议去掉没必要的<style>

**Pull request closed without merging**